release notes/v1.0.0-rc1.md

```diff
@@ -0,0 +1,186 @@
+
+For example, to track all requests and responses made by a page, you can use the following script:
+
+```javascript
+import { browser } from 'k6/browser';
+
+export const options = {
+  scenarios: {
+    ui: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+          type: 'chromium',
+        },
+      },
+    },
+  },
+};
+
+export default async function () {
+  const page = await browser.newPage();
+
+  // registers a handler that logs all requests made by the page
+  page.on('request', async request => console.log(request.url()));
+  // registers a handler that logs all responses received by the page
+  page.on('response', async response => console.log(response.url()));
+
+  await page.goto('https://quickpizza.grafana.com/', { waitUntil: 'networkidle' });
+  await page.close();
+}
+```
+
+Output might look like this:
+
+```bash
+INFO[0000] https://quickpizza.grafana.com/                  source=console
+INFO[0001] https://quickpizza.grafana.com/api/tools         source=console
+INFO[0001] https://quickpizza.grafana.com/images/pizza.png  source=console
+...
+```
+
+### `k6/experimental/webcrypto` promoted to stable and available globally under `crypto` [#4278](https://github.com/grafana/k6/pull/4278)
+
+With this release, the `k6/experimental/webcrypto` module is promoted to stable and available globally under `crypto`. So you don't need to import it anymore.
+
+```javascript
+export default function () {
+  const myUUID = crypto.randomUUID();
+
+  console.log(myUUID);
+}
+```
+
+`k6/experimental/webcrypto` is deprecated and will be removed in `v1.1.0`.
+
+### Support for custom templates in `k6 new` command [#4618](https://github.com/grafana/k6/pull/4618)
+
+The `k6 new` command now accepts a path to a file to use as a template for the new script. Templates use [Go templates syntax](https://pkg.go.dev/text/template#pkg-overview) and can include the following variables:
+
+- `ScriptName`: The name of the new script.
+- `ProjectID`: The ID of the Grafana Cloud project to use for the new script.
+
+To generate a new script using a custom template, use the following command:
+
+```sh
+k6 new --template /path/to/my-template.js
+```
+### Secret Sources [#4514](https://github.com/grafana/k6/pull/4514), [#4621](https://github.com/grafana/k6/pull/4621)
+
+Now k6 has a way to get secrets from different sources with a way to extend that. Among other things this means that the values received from a secret source will be redacted from the logs.
+
+At this point the two implementations included are to read them from key-value file or from cli flags, are meant mostly for testing the feature.
+
+It already has extension support that can be used to implement retrieving secrets from more secure sources.
+
+In the future k6 will likely add additional implementations that are more production ready.
```

Comment:
It's worth adding a usage and output example here.

Reply:
@mstoykov what's your opinion here?